src/Features/CSharp/Portable/Diagnostics/Analyzers/CSharpTypeStyleDiagnosticAnalyzerBase.cs

```diff
@@ -50,11 +50,7 @@ private void HandleVariableDeclaration(SyntaxNodeAnalysisContext context)
             var options = context.Options;
             var syntaxTree = context.Node.SyntaxTree;
             var cancellationToken = context.CancellationToken;
-            var optionSet = options.GetDocumentOptionSetAsync(syntaxTree, cancellationToken).GetAwaiter().GetResult();
-            if (optionSet == null)
-            {
-                return;
-            }
+            var optionSet = options.GetAnalyzerOptionSetAsync(syntaxTree, cancellationToken).GetAwaiter().GetResult();
```

Comment:
😕 Why are we not using `context.Options` directly? This option set should automatically contain the options from the IDE when invoked as a VSIX-installed analyzer, and omit those options when called via a NuGet-installed analyzer.

Reply:
I don't think I am following. The AnalyzerOptionSet provides the behavior you are describing. Are you saying the AnalyzerConfigOptions provides this behavior without needing to wrap the DocumentOptionSet?